### PR TITLE
movie: The Wild Blue Yonder (2005) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
Something unique among sci-fi films to be sure. Rating 4/5.

### CODE REVIEW

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 1]
```diff
@@ -5496,6 +5496,44 @@ mother and daughter [Isabel interacted with her mother, Jennifer.]
...
+:: References
+https://en.wikipedia.org/wiki/The_Wild_Blue_Yonder
+
+:: Ratings
+4 <paul>
+
+:: Major Themes
+extraterrestrial being [An unnamed extraterrestrial, who was indistinguishable from a human being in form, recounted how he traveled to Earth from the outer reaches of Andromeda with other members of his race to colonize the planet, but without success.]
+alien brought pandemic [A team of astronauts were sent on a mission to find a new home for humanity after scientists mistakenly reasoned that a deadly alien disease had started to proliferate around the Earth.]
+space travel [A team of astronauts astronauts were sent on a mission initially to the outer reaches of solar system in search of habitable planet, although they eventually seemed to have traveled much deeper into outer space.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:5526]: Typo: "astronauts astronauts"

Comment (odinlake) [reply notes/stories/film/film-scifi-2000s.st.txt:5526]: Also: _a_ habitable planet

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 2]
```diff
@@ -5496,6 +5496,44 @@ mother and daughter [Isabel interacted with her mother, Jennifer.]
...
+space colonization [The extraterrestrial narrator described how he and some of of people traveled to the outer reaches of the Andromeda galaxy to Earth to set up a colony, but without success. Humanity began a search for a new habitable planet to colonize after scientists mistakenly concluded that a deadly alien disease had started to proliferate around the Earth.]
+ocean planet [A team of astronauts discovered a glaciated ocean planet  and explored its depths. The extraterrestrial narrator's home world was an ocean planet located in the outer reaches of the Andromeda galaxy.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:5528]: Double spaces detected

**Review by odinlake — APPROVED** [review]
Typos galore!

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt:5526]: Whoops. Fixed up that little typo laden motivation.

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt:5528]: What can I say, Herzog memorized me with the re-contextualized archival footage of a 1989 space shuttle Atlantis mission.

**Pull request merged**